app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java

```diff
@@ -450,10 +450,7 @@ public void run() {
             case R.id.button_ok:
 
                 Bundle result = new Bundle();
-                result.putString(
-                        EXTRA_KEY_AREA,
-                        Double.toString(areaCalculator.getArea())
-                );
+
                 result.putString(
                         EXTRA_KEY_PERIMETER,
```

Comment:
Ah sorry ignore me, wasn't reading carefully. Definitely makes sense to add perimeter to the `EXTRA_KEY_RESPONSE_BUNDLE`